src/lakefs_spec/client_helpers.py

```diff
@@ -25,3 +27,33 @@ def commit(
     commit_creation = CommitCreation(message=message, metadata=metadata or {})
 
     client.commits_api.commit(repository=repository, branch=branch, commit_creation=commit_creation)
+
+
+def create_tag(client: LakeFSClient, repository: str, ref: str, tag: str) -> None:
+    tag_creation = TagCreation(tag, ref=ref)
+    client.tags_api.create_tag(repository=repository, tag_creation=tag_creation)
+
+
+def get_tags_and_commits(client: LakeFSClient, repository: str) -> dict:
+    return client.tags_api.list_tags(repository=repository)
```

Comment:
Why `_and_commits`? Does this list the tag commits too?

Reply:
Agree. Can rename this. The tags, however, always point to a specific commit. That is why I came up with the naming. But just `get_tags` is probably easier.